protos/gimbal/gimbal.proto

```diff
@@ -71,6 +80,15 @@ service GimbalService {
      rpc SubscribeControl(SubscribeControlRequest) returns(stream ControlResponse) {}
 }
 
+message SetAnglesRequest {
+    float pitch_deg = 1; // Pitch angle in degrees (negative points down)
+    float yaw_deg = 2; // Yaw angle in degrees (positive is clock-wise, range: -180 to 180 or 0 to 360)
+    float roll_deg = 3; // Roll angle in degrees
```

Comment:
Can we please change the order of the fields to match the usual MAVLink order: roll, pitch, yaw.

Reply:
Sure. Shall I do it, or do you do it?  As I said, this is my first time. Happy to help where I can.

Reply:
I suggest you change it. Thanks!